packages/lodestar/test/unit/api/impl/beacon/blocks/getBlock.test.ts

```diff
@@ -1,47 +1,20 @@
-import {BeaconBlockApi} from "../../../../../../src/api/impl/beacon/blocks";
+import sinon, {SinonStub} from "sinon";
 import * as blockUtils from "../../../../../../src/api/impl/beacon/blocks/utils";
-import sinon, {SinonStub, SinonStubbedInstance} from "sinon";
-import {BeaconChain, IBeaconChain} from "../../../../../../src/chain";
-import {ForkChoice} from "@chainsafe/lodestar-fork-choice";
 import {config} from "@chainsafe/lodestar-config/minimal";
-import {StubbedBeaconDb} from "../../../../../utils/stub";
 import {expect, use} from "chai";
 import chaiAsPromised from "chai-as-promised";
 import {generateEmptySignedBlock} from "../../../../../utils/block";
-import {BeaconSync} from "../../../../../../src/sync/sync";
-import {Network} from "../../../../../../src/network";
+import {BeaconBlockApi} from "../../../../../../src/api/impl/beacon/blocks";
 
 use(chaiAsPromised);
 
 describe("api - beacon - getBlock", function () {
-  const sandbox = sinon.createSandbox();
-
-  let blockApi: BeaconBlockApi;
-  let chainStub: SinonStubbedInstance<IBeaconChain>;
-  let dbStub: StubbedBeaconDb;
-  let forkChoiceStub: SinonStubbedInstance<ForkChoice>;
   let resolveBlockIdStub: SinonStub;
+  let blockApi: BeaconBlockApi;
 
-  beforeEach(function () {
-    forkChoiceStub = sinon.createStubInstance(ForkChoice);
-    chainStub = sinon.createStubInstance(BeaconChain);
-    chainStub.forkChoice = forkChoiceStub;
-    dbStub = new StubbedBeaconDb(sinon, config);
-    resolveBlockIdStub = sandbox.stub(blockUtils, "resolveBlockId");
-    blockApi = new BeaconBlockApi(
-      {},
-      {
-        chain: chainStub,
-        config,
-        db: dbStub,
-        network: sinon.createStubInstance(Network),
-        sync: sinon.createStubInstance(BeaconSync),
-      }
-    );
-  });
-
-  afterEach(function () {
-    sandbox.restore();
+  beforeEach(() => {
+    resolveBlockIdStub = this.ctx.sandbox.stub(blockUtils, "resolveBlockId");
```

Comment:
Is `this.ctx` fully typed? If so how does that work?

Reply:
Not really sure, honestly.  I just found this way to access the global test data, but didn't look too much under the hood of it yet.

Reply:
I'll look into though.